runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -966,6 +966,12 @@ class CompilationInfo
    uint32_t getLocalGCCounter() { return _localGCCounter; }
    void incrementLocalGCCounter() { _localGCCounter++; }
 
+#if defined(J9VM_INTERP_AOT_COMPILE_SUPPORT) && defined(J9VM_OPT_SHARED_CLASSES) && (defined(TR_HOST_X86) || defined(TR_HOST_POWER) || defined(TR_HOST_S390) || defined(TR_HOST_ARM))
```

Comment:
Do we need the `ifdef` here? I am in favor of eliminating these ifdefs as much as possible. The only place where we need them is when we use SCC APIs that only exist under such `ifdef` in the VM code.